tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -17,11 +17,56 @@
 #import "FunctionalTestHelpers.h"
 #import "UXTestHelpers.h"
 #import "UILabelInternal.h"
+#import "UIViewInternal.h"
 
 // Re-use existing sample code for validation
 #import "UIKitControls/UIButtonWithControlsViewController.h"
 
+#include "COMIncludes.h"
+#import <winrt/Windows.UI.Xaml.Media.Imaging.h>
+#include "COMIncludes_End.h"
+
 using namespace UXTestAPI;
+using namespace winrt::Windows::UI::Xaml;
+
+// Callback object for the CurrentBackgroundImage test.
+// Normally a closure can be created with a lambda, but a compiler issue prevents that in
+// this case. Until it's fixed, use this object as a workaround.
+class LayoutUpdatedCallback
```

Comment:
There's a compiler bug tracking this problem. Once the compiler is fixed, we can remove this workaround.